include/otf2xx/definition/marker.hpp

```diff
@@ -63,6 +63,8 @@ namespace definition
         {
         }
 
+        marker() = default;
```

Comment:
Just a question: AFAIR declaring a custom ctor disables the default ctor and `container` requires the object to be default constructible. There is a `using base::base;` but in the private section of the class. Hence I'd expect the base-ctors to be defined but private for `marker`.

So why does GCC7 compile this and why did you expect it to compile? What am I missing?

Ok, tried to move the `using base::base` but same, so it is only about the custom ctor disabling the default one.

Reply:
Yes and it's IMO the only correct way. Just wondered about your commit messages and whether there is something I missed. If you happen to find it, let me know.